### PR TITLE
Fix #5: Windows: Removable is always null; Refactor parser to use CSV format from wmic

### LINKED ISSUE #5: Windows: removable is always null
`wmic logicaldisk get caption,description,..,drivetype`

_DriveType_ is returned as an integer corresponding to the type of disk drive the logical disk represents:

0 = Unknown
1 = No Root Directory
2 = Removable Disk
3 = Local Disk
4 = Network Drive
5 = Compact Disc
6 = RAM Disk

### PR DESCRIPTION
**Pull request by jacargentina**

### CODE REVIEW

**Comment by jacargentina** [comment]
@philippefutureboy hoping your blessings. Leaving merge for you; Cheers!0

File: package.json [hunk 1]
```diff
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "bluebird": "^3.5.1",
+    "csv": "^3.0.2",
```
Comment (philippefutureboy) [inline package.json]: Is this extra dependency truly necessary?
Can't we split a csv simply by using something like `csv.split('\n').map((s) => s.split(',');`?

File: src/windows/windows.js [hunk 2]
```diff
@@ -1,46 +1,60 @@
...
-  acc.devices[name].whole = true;
-  acc.devices[name].parent = id;
-  acc.devices[name].node = caption;
-  acc.devices[name].name = name;
-  acc.devices[name].size = parseInt(size);
-  acc.devices[name].description = desc;
+export const parseWindowsProps = (
+  acc,
+  {
+    Caption,
```
Comment (philippefutureboy) [inline src/windows/windows.js]: Hmmm, personally style-wise I prefer to have parameter objects on a single line whenever possible. How many characters does it come up to? If you have a strong enough reason to put it that way, I don't mind moving forward with this. Let me know what you think

File: package.json (hunk 1 above)
Comment (jacargentina) [reply package.json]: I'll try to do the "mini parsing" (your code is very simple... its missing the column names for the objects, etc) but no problem!

File: src/windows/windows.js (hunk 2 above)
Comment (jacargentina) [reply src/windows/windows.js]: Sorry about that; i use sublime text with jsprettier in all my nodejs projects, thats the why (by default it cuts lines greater than 80, and other defaults about splitting lines on parameter objects, etc). But i can leave them on the same line if you want, no problem!

Comment (philippefutureboy) [reply src/windows/windows.js]: No worries, it happens :)

**Comment by philippefutureboy** [comment]
@jacargentina Can you merge master back in your feature branch and resolve conflicts?
After that we'll see how the coverage is holding up. Optimally we should maintain 98%+ coverage.

**Comment by philippefutureboy** [comment]
I still cannot merge back to master – There seems to be still some conflicts with the base branch.

**Comment by jacargentina** [comment]
@philippefutureboy Hope now its ready?

**Comment by philippefutureboy** [comment]
I'd love to say yes! :/

Can you verify if your change is responsible for the failed test on AppVeyor? If not, we'll merge and we'll open a new PR for the said test. Otherwise, can you correct the failing test?

Then we'll have to merge manually because it seems that there are some unresolved conflicts between your branch and master.

Keep up the good work, we'll make it happpeeeeeennnnn 🚀

**Comment by philippefutureboy** [comment]
All good! You can do the merge manually from your local repository. Cheers 🎉

**Pull request merged**